Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1609,14 +1610,6 @@ void WebProcess::prepareToSuspend(bool isSuspensionImminent, MonotonicTime estim
         platformMediaSessionManager->processWillSuspend();
 #endif
 
-#if !PLATFORM(MAC)
```

Comment:
Processes on iOS now no longer slim down on suspension and we thus have a higher background jetsam risk, as mentioned in my earlier review.

Reply:
Yes, that is a good point. Page load performance benefits from having this cached when the process is taken from the process cache. I wonder if we could delay releasing this memory until the very last assertion is dropped, since the process will be able to respond to memory pressure warnings before that?